novuhq/website
# feat: add Podcast pages

**Describe what changes this pull request brings**
This pull request adds the `Podcast` pages.


**Steps to test**
1. Open [the page](https://websitemain13551-podcast.gtsb.io/podcast/)
2. Confirm that everything looks and works as expected
